include-refactoring/xsimd/arch/xsimd_sse2.hpp

```diff
@@ -29,6 +29,23 @@ namespace xsimd {
       return _mm_add_pd(self, other);
     }
 
+    // all
+    template<class A> bool all(batch<float, A> const& self, requires<sse2>) {
+      return _mm_movemask_ps(self) == 0x0F;
+    }
```

Comment:
`_mm_movemask_ps` is available with SSE instruction set, therefore this should be moved back to `xsimd_sse.hpp`.